packages/phone-number-privacy/common/test/utils/sequential-delay.test.ts

```diff
@@ -0,0 +1,268 @@
+
+      const domain: SequentialDelayDomain = {
+        name: 'Sequential Delay Domain',
+        version: 1,
+        stages: [{ delay: t, batchSize: 2 }],
+      }
+
+      const attempts: TestAttempt[] = [
+        {
+          timestamp: t + 1,
+          expectedResult: {
+            accepted: true,
+            state: { timer: t + 1, counter: 1 },
+          },
+        },
+        {
+          timestamp: t + 1,
+          expectedResult: {
+            accepted: true,
+            state: { timer: t + 1, counter: 2 },
+          },
+        },
+      ]
+
+      checkTestAttempts(t, domain, attempts)
+    })
+
+    it('should accumulate quota when resetTimer is false', () => {
+      const t = 0 // initial delay
+
+      const domain: SequentialDelayDomain = {
+        name: 'Sequential Delay Domain',
+        version: 1,
+        stages: [
+          { delay: t, resetTimer: false },
+          { delay: 1, resetTimer: false },
+          { delay: 1, resetTimer: false },
+          { delay: 1, resetTimer: false },
+        ],
+      }
+
+      const attempts: TestAttempt[] = [
+        {
+          timestamp: t + 3,
+          expectedResult: {
+            accepted: true,
+            state: { timer: t, counter: 1 },
+          },
+        },
+        {
+          timestamp: t + 3,
+          expectedResult: {
+            accepted: true,
+            state: { timer: t + 1, counter: 2 },
+          },
+        },
+        {
+          timestamp: t + 3,
+          expectedResult: {
+            accepted: true,
+            state: { timer: t + 2, counter: 3 },
+          },
+        },
+        {
+          timestamp: t + 3,
+          expectedResult: {
+            accepted: true,
+            state: { timer: t + 3, counter: 4 },
+          },
+        },
+        {
+          timestamp: t + 3,
+          expectedResult: {
+            accepted: false,
+            state: { timer: t + 3, counter: 4 },
+          },
+        },
+      ]
+
+      checkTestAttempts(t, domain, attempts)
+    })
+
+    it('should not accumulate quota when resetTimer is true', () => {
+      const t = 0 // initial delay
+
+      const domain: SequentialDelayDomain = {
+        name: 'Sequential Delay Domain',
+        version: 1,
+        stages: [{ delay: t }, { delay: 1 }],
+      }
+
+      const attempts: TestAttempt[] = [
+        {
+          timestamp: t + 2,
+          expectedResult: {
+            accepted: true,
+            state: { timer: t + 2, counter: 1 },
+          },
+        },
+        {
+          timestamp: t + 2,
+          expectedResult: {
+            accepted: false,
+            state: { timer: t + 2, counter: 1 },
+          },
+        },
+        {
+          timestamp: t + 3,
+          expectedResult: {
+            accepted: true,
+            state: { timer: t + 3, counter: 2 },
+          },
+        },
+      ]
+
+      checkTestAttempts(t, domain, attempts)
+    })
+
+    it('Example', () => {
```

Comment:
Just following the ideomatic form for `it` clauses.
```suggestion
    it('should return he correct results in the example sequence', () => {
```